lib/matplotlib/colorbar.py

```diff
@@ -1395,7 +1394,7 @@ def make_axes(parents, location=None, orientation=None, fraction=0.15,
 
     Parameters
     ----------
-    parents : `~.axes.Axes` or sequence or `numpy.ndarray` of `~.axes.Axes`
+    parents : `~.axes.Axes` or iterable of `~.axes.Axes`
```

Comment:
Should this still include ndarray explicitly?  A 2D array can be passed (and gets flattened), and I’m not sure it counts as an iterable of Axes (rather an iterable of iterables of Axes…)

Sorry I missed that before.

Reply:
oh, yeah that is correct, putting it back.